.github/workflows/docs.yml

```
@@ -7,18 +7,87 @@ on:
     tags: '*'
   pull_request:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# We need spack so that we can install palace for the tutorials
+env:
+  SPACK_VERSION: develop
+  REGISTRY: ghcr.io/awslabs/palace
+  GITHUB_USER: ${{ github.actor }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
 jobs:
   build-docs:
     permissions:
+      packages: write
       contents: write
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
 
+      - name: Setup Spack
+        uses: spack/setup-spack@v2
+        with:
+          ref: develop
+          buildcache: true
+          color: true
+          # This seems unnecessary, but otherwise we clone into ./spack
+          # This then would override the spack local repo dir, and break CI
+          path: spack-path
```

With #427, we no longer need this with our internal spack packages repo changing it's pathname.

In addition to that, this workflow doesn't use that in the first place, so we don't need this.

```suggestion
          color: true
```

(I don't think GitHub is liking this sort of "only deleting lines" inline suggestion, so this might have to be manual)